### PR TITLE
Add new category: Backend-as-a-Service

### PR DESCRIPTION
**Pull request by fmerian**
https://appwrite.io/

## checklist

Please make sure you've complied with the [contribution guidelines](https://github.com/agamm/awesome-developer-first/blob/main/CONTRIBUTING.md):
- [x] The homepage clearly states that the product is for developers.
  > "Headless", "API-First", "SaaS" are frequently used keywords.
  > Usually this means that the front page has some code examples :)
- [x] The product is available now and requires payment.
- [x] The PR has a descriptive title -- e.g., `Add {Product}`, not `Update README.md`.
- [x] The new submission isn't a duplicate.
- [x] The entry has a description, is sorted alphabetically, and ends with a full stop.

Thanks!

### CODE REVIEW

**Comment by agamm** [comment]
Looks cool, but I'm not totally sure on the category, what are your thoughts @fmerian ?

**Comment by fmerian** [comment]
> Looks cool, but I'm not totally sure on the category, what are your thoughts @fmerian ?

sounds like Supabase, doesn't it?

**Comment by agamm** [comment]
Maybe we should create a new "Backend-as-a-Service" category?

**Comment by fmerian** [comment]
updated. ping @agamm

File: README.md [hunk 1]
```diff
@@ -93,6 +94,12 @@ For the latest additions [click here](https://github.com/agamm/awesome-developer
 * [n8n](http://n8n.io/) - Free and source-available fair-code licensed workflow automation tool. ![n8n](https://img.shields.io/github/contributors-anon/n8n-io/n8n?style=flat-square&logo=github&labelColor=%230D1117&color=%23161B22)
 * [Trigger.dev](http://trigger.dev/) - Open-source platform to create workflows from APIs, on a schedule or on demand. ![trigger.dev](https://img.shields.io/github/contributors-anon/triggerdotdev/trigger.dev?style=flat-square&logo=github&labelColor=%230D1117&color=%23161B22)
 
+## Backend-as-a-Service
+*Full set of tools for backend engineers.*
```
Comment (agamm) [inline README.md:98]:
```suggestion
*Abstract away all the backend hassles with an out-of-the-box suite of backend solutions.*
```

File: README.md [hunk 2]
```diff
@@ -93,6 +94,12 @@ For the latest additions [click here](https://github.com/agamm/awesome-developer
...
+* [Appwrite](https://appwrite.io/) - End-to-end backend server for frontend and mobile developers. ![Appwrite](https://img.shields.io/github/contributors-anon/appwrite/appwrite?style=flat-square&logo=github&labelColor=%230D1117&color=%23161B22)
+* [PlanetScale](https://planetscale.com/) - Hosted managed MySQL database, CI integration & auto scaling.
```
Comment (agamm) [inline README.md:100]: I think this is better in the database section along with neon.

File: README.md [hunk 3]
```diff
@@ -93,6 +94,12 @@ For the latest additions [click here](https://github.com/agamm/awesome-developer
 * [n8n](http://n8n.io/) - Free and source-available fair-code licensed workflow automation tool. ![n8n](https://img.shields.io/github/contributors-anon/n8n-io/n8n?style=flat-square&logo=github&labelColor=%230D1117&color=%23161B22)
 * [Trigger.dev](http://trigger.dev/) - Open-source platform to create workflows from APIs, on a schedule or on demand. ![trigger.dev](https://img.shields.io/github/contributors-anon/triggerdotdev/trigger.dev?style=flat-square&logo=github&labelColor=%230D1117&color=%23161B22)
 
+## Backend-as-a-Service
+*Full set of tools for backend engineers.*
+* [Appwrite](https://appwrite.io/) - End-to-end backend server for frontend and mobile developers. ![Appwrite](https://img.shields.io/github/contributors-anon/appwrite/appwrite?style=flat-square&logo=github&labelColor=%230D1117&color=%23161B22)
```
Comment (agamm) [inline README.md:99]: Internal tool maybe? 😅

**Review by agamm — APPROVED** [review]
Wrote my thoughts, but up to you @fmerian

File: README.md (hunk 3 above)
Comment (fmerian) [reply README.md:99]: idk, lgtm

**Pull request merged**